src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -191,6 +195,16 @@ def _accumulate_house_trades(house, grid, accumulated_trades, past_market_types)
                 seller_id = area_name_from_area_or_iaa_name(trade.seller)
                 accumulated_trades[house.name]["consumedFrom"][seller_id] += trade.offer.energy
                 accumulated_trades[house.name]["spentTo"][seller_id] += trade.offer.price
+
+    for market in getattr(house, past_market_types):
+        for trade in market.trades:
+            if area_name_from_area_or_iaa_name(trade.seller) == house.name:
+                accumulated_trades[house.name]["consumedFromExternal"] += trade.offer.energy
+                accumulated_trades[house.name]["spentToExternal"] += trade.offer.price
+            elif area_name_from_area_or_iaa_name(trade.buyer) == house.name:
+                accumulated_trades[house.name]["producedForExternal"] -= trade.offer.energy
+                accumulated_trades[house.name]["earnedFromExternal"] -= trade.offer.price
```

Comment:
This should be += and not -=, or did I not understand it correctly? The reason I am referring it is because it does not match the L190 of the same file.

Reply:
You are right! 
And additionally: I could change `-=` to `+=` one line above, too as these values are only needed once. Thus I can omit the `abs(` in L383 and L384 as well. 
Are you OK with that @spyrostz ?

Reply:
I think for the energy you need the signs, because the graph is going to plot one bar on the negative axis for produced energy, and on the positive axis for consumed energy. Therefore the signs have to remain for the energy and they cannot be avoided. It is just for the prices that need to be removed.

Reply:
You are right! Thanks! *:face_palm:*